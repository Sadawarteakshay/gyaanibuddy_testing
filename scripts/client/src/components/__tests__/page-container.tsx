import React from 'react';

import { render, cleanup } from '../../test-utils';
import PageContainer from '../page-container';
import renderer from 'react-test-renderer';

describe('Page Container', () => {
  // automatically unmount and cleanup DOM after the test is finished.
  afterEach(cleanup);

  it('renders without error', () => {
    render(<PageContainer />);
  });

  it('render snapshot', () => {
    const PageContainerElement = <PageContainer />;

    render(PageContainerElement);

    // Snapshot test - ensure button has no unsuspecting changes between commits
    const ElementTree = renderer
      .create(PageContainerElement)
      .toJSON();

    expect(ElementTree).toMatchSnapshot();
  });
});
