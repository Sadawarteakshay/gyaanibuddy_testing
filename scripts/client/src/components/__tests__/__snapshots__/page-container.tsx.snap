// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Container render snapshot 1`] = `
Array [
  <div
    className="css-k2u1cc"
  />,
  <div
    className="css-1on771l"
  />,
]
`;
